src/tests/base.h

```diff
@@ -7,6 +7,8 @@
 #include <memory>
 #include <string>
 
+#define UNUSED(x) (void)(x)
```

Comment:
I'd rather disable the warning

Reply:
Why? This makes it explicit.

Reply:
I don't see why it needs to be explicit, I have not come across a case where an argument was accidentally ignored (not saying it doesn't happen though). Ignoring it due to inheritance or for other reasons is however quite common.

Reply:
:thinking: right.